src/watchd/runtime.rs

```diff
@@ -85,8 +85,20 @@ impl WatcherRuntime {
         let msg = self.receiver.recv()?;
         debug!("Processing message {}", msg);
         trace!("Message details: {:?}", msg);
-        // TODO: Forward electrum notifications over the bridge
-        // self.send_over_bridge(msg.into()).expect("watcher bridge is halted");
```

Comment:
I think this should be uncommented and the block below should move to `From<...electrum_msg_type...> for CtlMsg`.

This is better since in this case other electrum messages wouldn't be lost

Reply:
> I think this should be uncommented and the block below should move to `From<...electrum_msg_type...> for CtlMsg`.
> 
> This is better since in this case other electrum messages wouldn't be lost

I tried uncommenting the line, but the conversion did not work. I can implement the `ElectrumUpdate` to `CtlMsg` conversion in another PR so it does not block the new release.

Reply:
There is a simpler way for now btw - see my comment below